model/card.go

```diff
@@ -14,6 +14,7 @@ type PhononCard interface {
 	OpenSecureConnection() error
 	Init(pin string) error
 	IdentifyCard(nonce []byte) (cardPubKey *ecdsa.PublicKey, cardSig *util.ECDSASignature, err error)
+	IdentifyPostedPhononNonce() (nonce uint64, err error)
```

Comment:
I like GetPostedPhononNonce() over Identify, since this is not an authenticating piece of data for a card, but that may just be a matter of taste, I'll leave this to your judgement.